packages/fileeditor-extension/src/commands.ts

```diff
@@ -943,6 +970,24 @@ export namespace Commands {
     });
   }
 
+  /**
+   * Add ___ File items to the Launcher for common file types associated with available kernels
+   */
+  export function addKernelLanguageLauncherItems(
+    launcher: ILauncher,
+    trans: TranslationBundle,
+    availableKernelFileTypes: Set<IExtensionData>
```

Comment:
You could make `availableKernelFileTypes` in each of these three functions be a `Iterable<IExtensionData>`, which would allow people to pass in any iterable, not just a Set.